extensions/azurePublish/src/components/azureProvisionDialog.tsx

```diff
@@ -659,119 +665,105 @@ export const AzureProvisionDialog: React.FC = () => {
-                label={formatMessage('Region for Luis')}
-                options={luisLocationsOption}
-                placeholder={formatMessage('Select one')}
-                selectedKey={formData.luisLocation}
-                onChange={(e, o) => {
-                  updateFormData('luisLocation', o.key as string);
-                }}
-              />
-            </form>
+      <form style={{ width: '100%' }}>
+        <Dropdown
+          ariaLabel={formatMessage(
+            'The Azure AD directory includes the tenant’s users, groups, and apps and is used to perform identity and access management functions for tenant resources.'
           )}
-          {formData.creationType === 'import' && (
-            <div style={{ width: '100%', height: '100%' }}>
-              <div
-                style={{
-                  fontSize: '14px',
-                  fontWeight: 600,
-                  color: '#323130',
-                  padding: '5px 0px',
-                }}
-              >
-                {formatMessage('Publish Configuration')}
-              </div>
-            </div>
+          disabled={allTenants.length === 1 || currentConfig?.tenantId}
+          errorMessage={loginErrorMsg}
+          label={formatMessage('Azure Directory')}
+          options={allTenants.map((t) => ({ key: t.tenantId, text: t.displayName }))}
+          selectedKey={formData.tenantId}
+          styles={{ root: { paddingBottom: '8px' } }}
+          onChange={(_e, o) => {
+            updateFormData('tenantId', o.key as string);
+          }}
+          onRenderLabel={onRenderLabel}
+        />
+        <Dropdown
+          required
+          ariaLabel={formatMessage('All resources in an Azure subscription are billed together')}
+          disabled={currentConfig?.subscriptionId}
+          errorMessage={subscriptionsErrorMessage}
+          label={formatMessage('Subscription')}
+          options={subscriptionOptions}
+          placeholder={formatMessage('Select one')}
+          selectedKey={formData.subscriptionId}
+          styles={{ root: { paddingBottom: '8px' } }}
+          onChange={(_e, o) => {
+            updateFormData('subscriptionId', o.key as string);
+          }}
+          onRenderLabel={onRenderLabel}
+        />
+        <ResourceGroupPicker
+          disabled={currentConfig?.resourceGroup}
+          newResourceGroupName={isNewResourceGroupName ? formData.resourceGroup : undefined}
+          resourceGroupNames={resourceGroupNames}
+          selectedResourceGroupName={isNewResourceGroupName ? undefined : formData.resourceGroup}
+          onChange={(choice) => {
+            updateFormData('resourceGroup', choice.name);
+            setErrorResourceGroupName(choice.errorMessage);
+          }}
+        />
+        <TextField
+          required
+          ariaLabel={formatMessage(
+            'This name will be assigned to all your new resources. For eg-test-web app, test-luis-prediction'
```

Comment:
What does this mean? (I know it's not your change)

Reply:
The provisioning code adds some prefixes to resource names that are cascading off this one.

Reply:
the "english" of it makes no sense is what I meant